src/fidesctl/core/apply.py

```diff
@@ -67,6 +67,34 @@ def echo_results(action: str, resource_type: str, resource_count: int) -> None:
     echo_green(f"{action.upper()} {resource_count} {resource_type} resource(s).")
 
 
+def validate_dataset_usage(taxonomy: Taxonomy) -> list:
```

Comment:
I think this should be `-> List[str]` for the return type?

Reply:
thanks! Tired eyes last night I guess when changing this around a bit, will fix 👍🏽